pkg/groups/handlers.go

```diff
@@ -25,10 +26,11 @@ import (
 const (
 	ROLE_TOKEN_KEY  = "roles"
 	GROUP_TOKEN_KEY = "groups"
+	API_KEY         = "groups"
 )
 
 type UpdateRolesRequest struct {
-	Roles []string `json:"roles" validate:"required"`
+	Roles []string `json:"roles" validate:"required,dive,required"`
```

Comment:
```suggestion
	Roles []string `json:"roles" validate:"required,dive"`
```

double required?or is it by design?